ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -37,10 +37,23 @@ $question-padding: 1.5rem;
         }
     }
 
+    .mat-accordion .mat-expansion-panel {
+        &:first-of-type, &:last-of-type {
+            border-radius: 0;
+        }
+    }
+
     .mat-expansion-panel-header,
     .mat-expansion-panel-content {
         font-size: 1.5rem;
         font-family: Untitled-Sans-Regular;
         letter-spacing: 0;
+        text-align: left;
+    }
+
+    .mat-expansion-panel-header.mat-expanded {
+        .faq-block__question {
+            font-family: Untitled-Sans-Medium;
```

Comment:
I'd recommend using variables for fonts, something like $primary-font-medium, since the fonts are global for all components in the study, better keep in one place for simple and fast replacement

Reply:
Good point. Fixed at 7c8055f26248f550dfcdc8e9260f6caf1ca2a76d